SnowballSwiftExample/SnowballSwiftExample/Example Views/SnowballSwiftExampleTitleLabelView.swift

```diff
@@ -11,9 +11,9 @@ import SnowballSwiftKit
 struct SnowballSwiftExampleTitleLabelView: View {
     var body: some View {
         VStack(alignment: .leading) {
-            SnowballTitleLabel(title: "Label", subtitle: "Provide additional context or helpful information.")
+            TitleLabel(title: "Label", subtitle: "Provide additional context or helpful information.")
             Divider()
-            SnowballTitleLabel(title: "Label")
+            TitleLabel(title: "Label")
```

Comment:
there is now something called labeled content:
https://developer.apple.com/documentation/swiftui/labeledcontent

and this may be the proper way to express this API:
https://developer.apple.com/documentation/swiftui/view/labeledcontentstyle(_:)

Reply:
the reasoning being that a "title label content" could be expressed via this api quite easily. it's not dissimilar from GroupBox.